charts/dapr/crds/subscription.yaml

```diff
@@ -51,6 +51,18 @@ spec:
                 type: string
               deadLetterTopic:
                 type: string
+              bulkSubscribe:
+                description: Represents bulk subscribe properies
+                properties:
+                  enabled: 
+                    type: string
```

Comment:
Should this be a boolean?

Reply:
I had considered everything string and then converted when actually required, but, it makes it better to consider the actual data types. Changed.